Marsiple/App/New Group/PhotoCollectionViewController.swift

```diff
@@ -11,13 +11,14 @@ import UIKit
 class PhotoCollectionViewController: UICollectionViewController {
     private let albumId: Int
     private var photos = [Photo]()
-    private let activityIndicatorView = UIActivityIndicatorView.autolayoutView()
     
     init(albumId: Int) {
         self.albumId = albumId
         super.init(collectionViewLayout: UICollectionViewFlowLayout())
         DataFetcher.getPhotos(albumId: albumId,
-                              success: { [weak self] photos in self?.photos = photos },
+                              success: { [weak self] photos in
+                                self?.photos = photos
+                                self?.collectionView?.reloadData() },
                               failure: { error in print(error.errorDescription) })
```

Comment:
I'm not feeling the formatting here, especially the `trailing }` from the `success` closure.

What do you think about: 
```
DataFetcher.getPhotos(albumId: albumId,
                       success: { [weak self] photos in
                           self?.photos = photos
                           self?.collectionView?.reloadData()
         }, failure: { error in print(error.errorDescription) })
```

Reply:
@dkulundzic This formatting is used throughout the app, so there would be a lot of places to change code. Also it would be strange to have two of the argument names lined up and the third one moved out of line.